vlan_and_lagg.rst Adjustments (#613)
opnsense/docs: source/manual/other-interfaces.rst

                                    disabling the lagg interface itself.
==================================  ==================================================================================================

.. Attention::

>     LAGG Proto, should be set to what ever your switch supports. Its best practice to use LACP if possible.

Reviewer: ```suggestion
    The LAGG protocol should match with the one your switch supports. It is best practice to use LACP if possible.
```

>     LAGG Proto needs to be set on both devices connecting the LAGG at the same value.  

Reviewer: ```suggestion
    Devices connected via LAGG require the same protocol.  
```

**LACP timeout**

LACP timeout has two modes; Slow/normal and Fast. It handles how fast the re-convergence occurs in case of link failure.

> This specifies how often are the heartbeats sent between the two LAGG connected devices

Reviewer: ```suggestion
This specifies how often the heartbeats are sent between the two LAGG connected devices.
```

==================================  =================================  =================================================================
LACP mode                           Heartbeat/Timeout interval         Enable
==================================  =================================  =================================================================
Slow/Normal                         30s                                Fast timeout turned off
Fast                                1s                                 Fast timeout turned on
==================================  =================================  =================================================================

> The Slow/normal is in most cases default.

Reviewer: ```suggestion
The Slow/normal timeout should be the default in most cases.
```

> In most scenarios keeping the LACP timeout on Slow/normal is preferable, due to vague vendor implementation of the LACP fast timeout it can cause connectivity disruption.

Reviewer: ```suggestion
Due to possible vendor implementation issues, keeping the LACP timeout on Slow/normal is preferable. Fast timeout can cause connectivity disruption in some cases.
```

> LACP timeout needs to be set on both devices connecting the LAGG at the same value, if not, heartbeats can be missed which will cause connectivity disruption. 

Reviewer: ```suggestion
LACP timeout requires the same value on both devices connected via LAGG. If not, heartbeats can be missed which will cause connectivity disruption.
```
